packages/x-components/static-docs/experience-search-and-discovery/product-results-ui.md

```diff
@@ -14,7 +14,7 @@ tailor the display to create the best product discovery experience for your shop
 elements and product-related information you want to include when displaying product results such as
 the price, a picture, or even an add-to-cart button.
 
-![X Components for results display](~@assets/media/overview_product_card.svg)
+![X Components for results display](~@assets/media/features/overview-product-results-card.svg)
```

Comment:
i see a patern in all the images being `~@assets/media/ineterface/` but this is `~@assets/media/features/`, is that correct?

Reply:
yes, it's a shared asset. It's owned from the features/overview docs and reused inhere :)